textures/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'java'
+    id("com.github.johnrengelman.shadow") version "6.1.0"
+}
+
+group 'com.dumbdogdiner'
+version '2.2.0'
+
+repositories {
+    mavenCentral()
+    maven {
+        name("headdatabase-api")
+        url("https://mvn.intellectualsites.com/content/repositories/thirdparty/")
+    }
+    maven { url 'https://hub.spigotmc.org/nexus/content/repositories/public/' }
+}
+
+dependencies {
+    compile project(":common")
+    testCompile project(":common").sourceSets.test.output
+
+    compile project(":webapi")
+    testCompile project(":webapi").sourceSets.test.output
+
+    compileOnly("me.arcaniax:HeadDatabase-API:1.1.0")
+
+    implementation("org.jsoup:jsoup:1.13.1")
+    implementation(platform("com.squareup.okhttp3:okhttp-bom:4.9.0"))
+
+    implementation("com.squareup.okhttp3:okhttp")
+    implementation("com.squareup.okhttp3:logging-interceptor")
+    implementation 'commons-validator:commons-validator:1.7'
```

Comment:
Cloned from previous PR:
![image](https://user-images.githubusercontent.com/12949077/115428509-5da62e80-a1fa-11eb-8145-a37935edc808.png)

Reply:
CommonsValidator also allows us to remove a bunch of the luhn/shortid code we have; I can write a URL regex if necessary.

Reply:
`https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)` here's one :3

Reply:
Would be very appreciated if you can stick with Preconditions since we can technically use it.

Reply:
I wrote a special one, and it should be in there, not sure if pushed. Included domain checking